tests/_ast/codegen_data/test_generate_filecontents_async.py

```diff
@@ -0,0 +1,30 @@
+import marimo
+
+__generated_with = "0.1.0"
+app = marimo.App()
+
+
+@app.cell
+def one():
+    import numpy as np
+    import asyncio
+    return asyncio, np
+
+
+@app.cell
+async def two(asyncio):
+    x = 0
+    xx = 1
+    await asyncio.sleep(1)
+    return x, xx
+
+
+@app.cell
+def three(asyncio, x):
+    async def _():
+        await asyncio.sleep(x)
+    return
```

Comment:
should we test unparsable async code too? i assume it doesnt add the async, but still a good case to cover

Reply:
also maybe a test case for when someone adds await, without async (e.g. they manually edited the file), would it recorrect itself?

Reply:
Added a test for the first comment. For the second comment, the module fails to load with a syntax error (`await` has to be used inside an `aync` function, unless a special compiler flag is set), so didn't add a test for that.